deploy.js

```diff
@@ -37,8 +37,8 @@ function spawn(config, args, cb) {
     }).join(' ');
     cmd = [cmd, cmdArgs].join(' ');
   }
-
-  var proc = child_process.spawn('sh', ['-c', cmd], { stdio: 'inherit' });
+  cmd = cmd.split(' ');
```

Comment:
Splitting arguments will still lead to `unfunctionalities` like when a `dir` like `"test test"` is supplied